libdevcore/Exceptions.cpp

```diff
@@ -27,7 +27,9 @@ char const* Exception::what() const noexcept
 	if (string const* cmt = comment())
 		return cmt->c_str();
 	else
```

Comment:
You can skip `else`. Especially if the statement attached to `else` is so far away.

Reply:
Which one is far?

Reply:
`return std::exception::what();` is 3 line below `else`.